exercises/grains/src/test/java/GrainsTest.java

```diff
@@ -0,0 +1,99 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class GrainsTest {
+
```

Comment:
I would prefer a non-static implementation. See for example the pangram exercise.
This is policy, sorry I missed it in my previous comments.

Reply:
No problem. I will change this to a standard class.